.travis.yml

```diff
@@ -19,8 +20,6 @@ jobs:
       install: gem install awesome_bot
       script:
         - find . -type f -name '*.md' -exec awesome_bot {} \;
-      notifications:
-        email: false
```

Comment:
this actually doesn't do anything, maybe it was feature that was deprecated a while back, so removing for clarity